src/analysis/schwartz_space.lean

```diff
@@ -0,0 +1,383 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+
+import analysis.calculus.cont_diff
+import analysis.complex.basic
+import analysis.locally_convex.with_seminorms
+import topology.algebra.uniform_filter_basis
+import tactic.positivity
+
+/-!
+# Schwartz space
+
+This file defines the Schwartz space and the space of tempered distributions. Usually,
```

Comment:
There are no distributions yet in this space, right?

Reply:
They are automatically defined as the continuous dual space, but you might argue that it is a bit shaky since we don't have a topology on that space. But in principle you could define the delta-distribution.

Reply:
this here works:
````
def delta {x₀ : E} : 𝓢(E, ℂ) →L[ℂ] ℂ :=
{ to_fun := λ f, f x₀,
  map_add' := λ f g, by simp,
  map_smul' := λ a f, by simp,
  cont := sorry }
````
to fill in the continuity sorry one needs either to use the less-then-optimal things in `with_seminorms` or write some stuff essentially about Fréchet spaces.

Reply:
Maybe remove the reference to distributions for now, since there is nothing explicit in this file about them.

Reply:
ok, done